server/services/homekit/lib/buildService.js

```diff
@@ -68,6 +68,15 @@ function buildService(device, features, categoryMapping) {
         });
         break;
       }
+      case `${DEVICE_FEATURE_CATEGORIES.MOTION_SENSOR}:${DEVICE_FEATURE_TYPES.SENSOR.BINARY}`: {
+        const motionCharacteristic = service.getCharacteristic(Characteristic.MotionDetected);
+
+        motionCharacteristic.on(CharacteristicEventTypes.GET, async (callback) => {
+          const { features: updatedFeatures } = await this.gladys.device.getBySelector(device.selector);
+          callback(undefined, +!updatedFeatures.find((feat) => feat.id === feature.id).last_value);
```

Comment:
In sendState file you did not put `+!` to inverse state. I added it for opening sensor because state is inverted between Homekit & Gladys, is it necessary for motion?

Reply:
I will test tomorrow with a device. Any way to know before?

Reply:
Fixed